part-spec/common/thermals.json

```diff
@@ -0,0 +1,81 @@
+{
+    "$id": "https://github.com/edatasheets/digital-datasheets/blob/main/part-spec/common/thermals.json",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "specification of thermals ",
+    "thermalSpec": {
+        "type": "object",
+        "properties": {
+            "junctionTemp": {
+                "description": "recommended operating junction temperature range of a part",
+                "comment": "units of Celsius",
+                "$ref": "../common/values.json#/valueOptions"
+            },
+            "junctionTempMax": {
+                "description": "absolute maximum junction temperature range of a part",
```

Comment:
is this a range or just a single temperature?

Reply:
It is a range